actions/kubebuilder/entrypoint.sh

```diff
@@ -8,6 +8,8 @@ curl -sL https://go.kubebuilder.io/dl/${VERSION}/linux/amd64 | tar -xz -C /tmp/
 
 mkdir -p $GITHUB_WORKSPACE/kubebuilder
 mv /tmp/kubebuilder_${VERSION}_linux_amd64/* $GITHUB_WORKSPACE/kubebuilder/
+# remove kubectl as this is supposed to be installed separately
```

Comment:
Wouldn't it be better to remove this before doing `mv`?

Reply:
you're right. Fixed.